client/css/style.css

```diff
@@ -67,7 +67,7 @@ h3 {
 }
 
 h1.title {
-	margin-bottom: 10px;
+	margin: -30px 0 -20px 0;
```

Comment:
Can this be fixed in a way that doesn't require negative margins?

Reply:
Sure, I have removed the padding top of the element that was setting it rather than having negative margin that was effectively just negating it.